api/endpoints/activities.py

```diff
@@ -1,11 +1,56 @@
 """Module for Activities in Andela."""
-from flask import jsonify
+from flask import jsonify, request
 from flask_restplus import Resource
 
 from api.utils.auth import token_required
 from api.models import Activity
 
 
 class ActivitiesAPI(Resource):
-    """Activity Resource."""
-    pass
+    """Activity Resource to contain
+    CRUD endpoints for activities"""
+
+    @token_required
+    def post(self):
+        """ Create an activity """
+
+        payload = request.get_json()
+
+        if not payload:
+            response = jsonify({
+                "status": "fail",
+                "message": "name and description required"})
+            response.status_code = 400
+        elif "name" not in payload.keys():
```

Comment:
Also, what happened to using marshmallow?

Reply:
@LarryWachira I wanted to make sure that the endpoint implementation works as it should first then go ahead and create a schema for this. I will now be creating it.

Reply:
@LarryWachira I have completed adding marshmallow. I have included what you had worked on as a base then added my implementation. You can review then provide feedback.